tests/settings.spec.ts

```diff
@@ -9,23 +9,31 @@ describe('Settings', () => {
 
   it('should make a real network call if network is enabled and an unmocked request is fetched', async () => {
     require('netmock-js').allowRealNetwork(true);
-    await fetch('https://wix.com');
-    await axios.get('https://wix.com');
-    await expect((global as any).fetchSpy).toHaveBeenNthCalledWith(1, 'https://wix.com', undefined);
-    await expect((global as any).fetchSpy).toHaveBeenCalledTimes(2);
+    await expect(async () => {
+      await fetch('https://this-site-does-not-exist-niryo.com');
+    }).rejects.toThrowError('ENOTFOUND');
```

Comment:
```suggestion
    await expect(fetch('https://this-site-does-not-exist-niryo.com')).rejects.toThrowError('ENOTFOUND');
```
wouldn't this be equivilent?

Reply:
checking

Reply:
you are right, it works. pushing a fix now